intro-HPC/ch_MATLAB.tex

```diff
@@ -0,0 +1,80 @@
+\chapter{MATLAB}
+\label{ch:matlab}
+
+\section{Why is the MATLAB compiler required?}
+
+The main reason behind this alternative way of using MATLAB is licensing: only
+a limited number of MATLAB sessions can be active at the same time. However, once
```

Comment:
elaborate on this (under `\ifgent`): a limited number of MATLAB session can be active at the same time before there are only a limited number of MATLAB research licenses available on the UGent-wide MATLAB license server. Combine that with the abundance of compute resources in the HPC-UGent infrastructure, and you have a problem. (rephrase this a bit, obviously ;-))